src/components/books/Books.js

```diff
@@ -0,0 +1,7 @@
+export default function Books() {
+  return (
+    <div>
+      <h3>All Books</h3>
+    </div>
```

Comment:
## [OPTIONAL]
* Sorry this was not pointed out by the previous reviewer that is why this suggestion is marked optional but i would strongly suggest you implement them as it is a requirement needed to move to the next milestone 
* It'd be great if Books.js show a list of books, please work on it. 🐶
 Please, create a book component for each book, these books could be listed on books.js according to the project requirements.